charts/tekton-pipelines/values.yaml

```
@@ -66,7 +66,11 @@ buildpacks:
 
             [ -f package.json ] && {
               echo "install node.js dependencies"
-              npm ci --silent
+
+              # set cache
+              npm config set cache $(workspaces.source.path)/.npm --global
+              npm ci
+
               (npm run build:$(params.environment) && rm -rf node_modules && echo "remove node_modules dir") || (echo "unable to build $(params.environment)" && exit 1)
```

```suggestion
              (npm run build:$(params.environment) && rm -rf node_modules && echo "remove node_modules dir") || (echo "unable to build $(params.environment) exit_code: $?" && exit 1)
```
Exit code can help us to debug silent errors like this: https://github.com/nodejs/docker-node/issues/1734